packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3152,6 +3169,47 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   }
 
+  async seekRow(inputString: string, isBackwards = false): Promise<void> {
+    const {
+      gotoValueSelectedColumn: selectedColumn,
+      gotoValueSelectedFilter,
+      lastSeekedRow,
```

Comment:
I think instead of `lastSeekedRow`, we should use the users current selection for seeking. If it's forward, go from the last row in their selection, if it's backward, go from before the first row in their selection.